suite-common/graph/src/graphDataFetching.ts

```diff
@@ -110,7 +110,12 @@ const getBalanceFromAccountInfo = ({
             if (contractId) {
                 const token = accountInfo.tokens?.find(t => t.contract === contractId);
 
-                return token?.balance || '0';
+                if (token && token.balance) {
+                    // this is raw value from getAccountInfo, we need to divide it by 10^decimals (in redux it's already formatted)
```

Comment:
Wouldn't be better for consistency to have token.balance in redux in base unit without decimals?